angular-portfolio/src/app/resTemplate.ts

```diff
@@ -0,0 +1,5 @@
+export interface Res {
```

Comment:
I'd recommend keeping file name convention consistent. Elsewhere you are using dashes to separate words in filenames, but here it's camel cased. I think it should be res-template.ts